broker/lib/fabrik/Fabrik.js

```diff
@@ -76,4 +76,7 @@ Fabrik.FabrikStatusPoller = FabrikStatusPoller;
 Fabrik.dbManager = new DBManager();
 Fabrik.BoshTaskPoller = BoshTaskPoller;
 Fabrik.oobBackupManager = OobBackupManager;
+if (config.enable_service_fabrik_v2) {
+  Fabrik.UnlockResourcePoller = require('./UnlockResourcePoller');
```

Comment:
I would prefer to have this check in the `UnlockResourcePoller` itself rather than here. I am guessing this was done to prevent the load for both scheduler job as well as in case the flag is turned off. This file is meant as a "bastion" for referencing other modules- conditional references should be avoided as much as possible. Please think over it though